README.md

```diff
@@ -18,17 +18,26 @@ minus is an asynchronous terminal [paging] library written in Rust.
 </p>
 
 ## Features
-- Send data as well as configure the pager on the fly
+- Send data as well as configure the pager on the fly.
+    This means that your data can be shown on the pager's screen as soon as it is loaded by your application. But not only that,
+    you can also configure the minus while its running.
```

Comment:
```suggestion
- Send data and configure the pager while it's running
```

Reply:
What's the problem with that para?

Reply:
it's in my opinion too long. I wouldn't fully read the paragraph and this sentence includes all necessary information in my opinion

Reply:
Ah! Actually GitHub is merging the lines, I actually wanted them to be separate paragraphs. Like the first line should be as it is and following lines are a bit indented. Tomorrow I will see if I can get that or just remove that entire.